darts/models/forecasting/forecasting_model.py

```diff
@@ -1483,10 +1483,28 @@ def gridsearch(
             logger,
         )
 
+        if not isinstance(parameters, dict):
+            raise_log(
+                ValueError(
+                    f"`parameters` should be a dictionary, received a: {type(parameters)}."
+                )
+            )
+
+        if not all(isinstance(params, list) for params in parameters.values()):
```

Comment:
Should we allow np arrays as well?
```suggestion
        if not all(isinstance(params, (list, np.ndarray)) for params in parameters.values()):
```

Reply:
Sure, I restricted it to list because it's the type explicitly mentioned in the docstring.